apps/tailwind-components/components/Pagination.vue

```diff
@@ -64,12 +84,26 @@ function changeCurrentPage(event: Event) {
     <span :id="`${pageInputId}Label`" class="sr-only">
       pagination navigation
     </span>
-    <ul class="flex items-center justify-center list-none">
+    <ul class="flex items-center justify-center list-none gap-[10px]">
```

Comment:
Not a major issue: For `gap-[10px]`, I've started using (and converting all instances to) the tw class `gap-2.5`. Might be good to make a quick change while we are here.